android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java

```diff
@@ -84,20 +90,20 @@ public void onPermissionResult() {
     private boolean hasSettingsPermission() {
         return Build.VERSION.SDK_INT < Build.VERSION_CODES.M ||
                 (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M &&
-                Settings.System.canWrite(getReactApplicationContext()));
+                        Settings.System.canWrite(getReactApplicationContext()));
```

Comment:
You can likely simplify this with `ContextCompat`

https://github.com/robinpowered/robin-compass/blob/779ed696f8d09cbf586a16119011a257dc883fd3/android/app/src/main/java/com/robinpowered/compass/reactnative/PermissionManager.java#L168